src/main/kotlin/org/rust/lang/core/macros/MacroExpansionTask.kt

```diff
@@ -86,12 +90,12 @@ abstract class MacroExpansionTaskBase(
         try {
             submitExpansionTask()
 
-            MACRO_LOG.trace("Awaiting")
-            val millis = measureNanoTime {
+            MACRO_LOG.info("Awaiting")
```

Comment:
Let's change log levels in other PR. (I don't think it's reasonable to log "Awaiting" as INFO, and other messages should be also improved)